server/pages.js

```diff
@@ -196,4 +196,11 @@ pages.add(
   'new-collective-page',
 );
 
+// New accept financial contributions flow
+
+pages.add(
+  'accept-financial-contributions',
+  '/:slug/contribute/accept/:path(myself|organization|host)?/:method(stripe|bank)?/:state(success)?',
```

Comment:
Even if it's long it should be `accept-financial-contributions` because `contribute/accept` is not really clear.

Reply:
👍  Will change